### DISCUSSION TITLE
Undercurl

### DISCUSSION THREAD

**Discussion by p-kolacz** [arcticicestudio/nord#216] [Help & Support]
Is it possible to change underline for SpellBad to undercurl?

**Comment by arcticicestudio** [comment]
Hi @p-kolacz 👋🏼 

You can customize all syntax highlighting groups set by Nord in your configuration. Please see the [documentation about how to customize Nord][d] for more details.

[d]: https://www.nordtheme.com/docs/ports/vim/customization

**Comment by svengreb** [comment]
@p-kolacz Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.

**Comment by yochem** [comment]
You could add it to your config like this:

vimscript:
```vim
highlight SpellBad gui=undercurl cterm=undercurl
```
